index.html

```diff
@@ -3,13 +3,26 @@
 
 <head>
   <meta charset="utf-8">
-   
+  
   <title>Sahil Jaiswal</title>
+  <link rel="stylesheet" href="style.css">
+   
 </head>
+
+<body>
+  <div>
+    <a style="color: black" href="#">Resume</a>
+  </div>
+  <div>
+    <h1 class="name">Sahil Jaiswal</h1>
+  </div>
+  <div class="profile">
+    <img style="padding-left: 70%" src="shivam-prifile.png" alt="Profile" height="300px">
```

Comment:
in src column, attach my image link "https://avatars.githubusercontent.com/u/43785914?v=4"